.github/workflows/nix.yml

```diff
@@ -0,0 +1,22 @@
+name: "CI - Nix"
+
+on:
+  push:
+    branches:
+    - humble_devel
+  pull_request:
+    branches:
+    - humble_devel
+
+jobs:
+  nix:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: cachix/install-nix-action@v30
+      - uses: cachix/cachix-action@v15
+        with:
+          name: gepetto
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+          extraPullNames: ros
+      - run: nix build -L .#agimus-controller-examples .#agimus-controller-ros
```

Comment:
Sorry I forgot to ask before merging: why not `.#agimus-controller` ? in addition to the others?
If I am correct this way you do not run the `agimus_controller` unit-test, isn't it?

Reply:
It is a dependency of the others, so it is built in any case.
This still run unit tests, as the package define that tests must be run.
Except there are no tests. Or did I missed those ?

Reply:
they on the topic/humble-devel/refactor branch that will be merge into here.